CentralComputing/Pod.cpp

```diff
@@ -29,35 +38,103 @@ void int_handler(int signum) {
 int pod(int argc, char** argv) {
 	
 	signal(SIGINT, int_handler);
```

Comment:
We should use `sigaction(2)` instead

Specifically from the `signal(2)` [man page: ](https://linux.die.net/man/2/signal) 

> The effects of signal() in a multithreaded process are unspecified.

To avoid something strange down the road, even though this method 'works'